README.md

```diff
@@ -30,3 +30,14 @@
 3. [sass-loader](https://www.npmjs.com/package/sass-loader)
 4. [Storybook](https://storybook.js.org/docs/guides/guide-react/)
 5. [Enzyme](https://enzymejs.github.io/enzyme/)
+
+### E. Deployments
+- This application is deployed on heroku, we have two different versions, which are different by the app status, in staging mode or ready for production.
+
+- When the app is in staging mode is available on this link `https://stackup-barefoot.herokuapp.com/` => This deployment happens automatically, whenever there is a new merging to develop branch
+
+- When the app is ready for production is available on this link `https://immense-chamber-27847.herokuapp.com/`
```

Comment:
`immense-chamber-27847`

Reply:
> `immense-chamber-27847`

it's the auto-generated name for the project on Heroku in production because I didn't provide the name for the project